### PR TITLE
Release/1.0.0 - solution-pack-fortimanager-ztp-feature-examples

### PR DESCRIPTION
**Pull request by fortijames**
Please review and merge for release.

This does require the pending pull request of the new FortiManager ZTP Flow Solution Pack ( https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-framework/pull/2 ).

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    "dependencies": [
+        {
+            "name": "fortiManagerZTPFlow",
+            "type": "solutionpack",
+            "label": "FortiManager ZTP Flow",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Community",
```
Comment (cs-rohit-khune) [inline info.json]: Update publisher to "Fortinet CSE"

**Review by cs-rohit-khune — CHANGES_REQUESTED** [review]

**Comment by cs-rohit-khune** [comment]
@fortijames Since this solution pack only includes demo records, it's recommended to use a scenario to create the same. [Refer to https://fortisoar.contenthub.fortinet.com//detail.html?entity=socSimulator&version=1.0.2&type=solutionpack]

**Comment by cs-rohit-khune** [comment]
Please exclude the "modules" folder from the solution pack, as the "FortiManager ZTP Flow" pack already provides these modules.

File: info.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    "postInstallConfig": null,
+    "recordTags": [
+        "FeatureTest",
+        "ZTP",
+        "FortiManager"
+    ],
+    "date": "2023-07-13T19:53:09+00:00",
+    "contents": {
+        "modules": {
+            "attachments": {
```
Comment (cs-rohit-khune) [inline info.json]: Please remove all the modules

**Comment by fortijames** [comment]
> @fortijames Since this solution pack only includes demo records, it's recommended to use a scenario to create the same. [Refer to https://fortisoar.contenthub.fortinet.com//detail.html?entity=socSimulator&version=1.0.2&type=solutionpack]

I don't know if this will work very well for this framework. These records are not SOC Simulations. I might need some help going through what this would look like using the SOC Simulator solution pack.

**Review by cs-suyog-jinturkar — APPROVED** [review]
Reviewed by Rohit. Approving here.

**Pull request merged**